README.md

```diff
@@ -101,6 +101,22 @@ that iterates through them:
 order.calculate_total_price(double(:price => 1.99), double(:price => 2.99))
 ```
 
+### Stubbing a chain of methods
+
+You can use `receive_message_chain` in place of `receive` in certain circumstances to stub a chain of messages:
+
+```ruby
+allow(double).to receive_message_chain("foo.bar") { :baz }
+allow(double).to receive_message_chain(:foo, :bar => :baz)
+allow(double).to receive_message_chain(:foo, :bar) { :baz }
+```
+
+Given any of these three forms:
```

Comment:
Usually when I see a sentence like this, I expect the 3 forms to follow (it seems like it is introducing the 3 forms)....but it's not.

Might make more sense to just merge this snippet with the above:

``` ruby
# ...
allow(double).to receive_message_chain(:foo, :bar) { :baz }

# Given any of the above forms:
double.foo.bar # => :baz
```